test/unit/fiber_channel.cc

```diff
@@ -18,9 +18,9 @@ fiber_channel_basic()
 
 	ok(fiber_channel_count(channel) == 0, "fiber_channel_count()");
 
-	ok(fiber_channel_is_full(channel) == false, "fiber_channel_is_full()");
+	ok(!fiber_channel_is_full(channel), "fiber_channel_is_full()");
 
-	ok(fiber_channel_is_empty(channel) == true, "fiber_channel_is_empty()");
+	ok(fiber_channel_is_empty(channel), "fiber_channel_is_empty()");
```

Comment:
> Don't do a change if it is not related to the patch.

https://github.com/tarantool/tarantool/wiki/Code-review-procedure#general-coding-points-to-check